src/build_helper/lib.rs

```diff
@@ -40,7 +40,9 @@ pub fn gnu_target(target: &str) -> String {
 }
 
 pub fn cc2ar(cc: &Path, target: &str) -> PathBuf {
-    if target.contains("musl") || target.contains("msvc") {
+    if target.contains("msvc") {
+        PathBuf::from("lib")
```

Comment:
Can this change be saved for later? Papering over this unfortunately probably won't fix much, the syntax for calling `lib` is very different than `ar`, so this will never work if it's actually used. This should basically just return an option and then it's handled elsewhere when it's not available